Gemfile

```diff
@@ -131,6 +130,7 @@ group :development, :test do
   gem 'factory_girl_rails'
   gem 'rspec-rails'
   gem 'rspec_junit_formatter'
+  gem 'rails-controller-testing'
```

Comment:
__[RuboCop]__

Bundler/OrderedGems: Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem `rails-controller-testing` should appear before `rspec_junit_formatter`.


<!-- Comment by [SideCI](https://sideci.com) -->